PLP/functions_1_prime.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+
+def prime(num):
```

Comment:
the algorithm is good, but how would you make use of python "all" statement in order to rewrite the function as one liner